docs/source-pytorch/fabric/guide/multi_node/barebones.rst

```diff
@@ -0,0 +1,161 @@
+:orphan:
+
+##################
+Bare Bones Cluster
+##################
+
+**Audience**: Users who want to train on multiple machines that aren't part of a managed cluster.
+
+This guide shows how to run a training job on a general-purpose cluster.
+It assumes that you can log in to each machine and run commands.
+
+Don't want to maintain your own infrastructure? Try the :doc:`Lightning cloud <./cloud>` instead.
+
+
+----
+
+
+************
+Requirements
+************
+
+To set up a multi-node computing cluster, you need the following:
+
+1. Multiple computers with Lightning installed
+2. A network connectivity between the machines with firewall rules that allow traffic flow on a specified port.
+
+|
```

Comment:
what does this do? :thinking:

Reply:
It adds a bit of space between the two paragraphs. Otherwise, the text below really sticks close to the bullet points